galley/pkg/config/analysis/msg/messages.yaml

```diff
@@ -34,12 +34,14 @@ messages:
     code: IST0102
     level: Info
     description: "A namespace is not enabled for Istio injection."
-    template: "The namespace is not enabled for Istio injection. Run 'kubectl label namespace %s istio-injection=enabled' to enable it, or 'kubectl label namespace %s istio-injection=disabled' to explicitly mark it as not needing injection."
+    template: "The namespace is not enabled for Istio injection. Run 'kubectl label namespace %s istio-injection=enabled' to enable it, or 'kubectl label namespace %s istio-injection=disabled' to explicitly mark it as not needing injection. You can also unset 'istio-injection' by running 'kubectl label namespace %s istio-injection-'"
```

Comment:
Why do we need info to unset istio-injection on a check that is only triggered when istio-injection is not set?